libs/shared/src/consts/providers/channels/email.ts

```diff
@@ -98,6 +90,14 @@ export const emailProviders: IProviderConfig[] = [
     docReference: 'https://netcorecloud.com/email/email-api/',
     logoFileName: { light: 'netcore.png', dark: 'netcore.png' },
   },
+  {
```

Comment:
How is the backward compatibility for those that are already using nodemailer when you change the id?